providers/scaleway/flags.go

```diff
@@ -0,0 +1,81 @@
+package scaleway
+
+import (
+	"os"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/urfave/cli/v2"
+)
+
+const category = "Scaleway"
+
+//nolint:mnd
+var ProviderFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:     "scaleway-access-key",
+		Usage:    "Scaleway IAM API Token Access Key",
+		EnvVars:  []string{"WOODPECKER_SCW_ACCESS_KEY"},
```

Comment:
Would also suggest to use `WOODPECKER_SCALEWAY_`

```suggestion
		EnvVars:  []string{"WOODPECKER_SCALEWAY_ACCESS_KEY"},
```